### PR TITLE
Added MIT License File

### LINKED ISSUE #863: Provide a License
Hey,

would you please provide a license for this project so it can be safely used in projects?

Thanks in advance.

### PR DESCRIPTION
**Pull request by MuzafferDede**
Fixes #863 
(Not sure if it's intentional that License.md is missing) 
> Is this something that is wanted/needed? Did you create a feature-request issue first?
**Yes.**

> Does it contain multiple, unrelated changes? Please separate the PRs out.
**No.**

> Does it include tests if possible? (Not a deal-breaker, just a nice-to-have)
**No need.**

> Please include a thorough description of the feature/fix and reasons why it's useful.
**It needs to be clear which license it has so livewire can be safely used in projects.**

> Thanks for contributing! 🙌
**Thank you too!**

### CODE REVIEW

**Comment by devcircus** [comment]
I've never seen a license that included "contributors" in the copyright. I'm sure there are examples but I'd think it should just be Caleb. 🤷🏻‍♂️ Also, it would be good to include a section at the bottom of the Readme that links to the license file.

**Comment by DanHarrin** [comment]
Also, not sure it's wise to include the year on the license or not, we don't exactly have the luxury of `date('Y')` here so could be a pain to update 😉

**Comment by roni-estein** [comment]
I never noticed. But technically a project without a license is illegal to use. Not normally a huge deal. But I’ve started using this for clients. I hope you settle one one shortly.
  

  
  
  

  
  
>

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## MIT License
+
+Copyright © 2019-2020 Caleb Porzio and contributors
```
Comment (DanHarrin) [inline LICENSE.md:3]:
```suggestion
Copyright © Caleb Porzio
```

File: README.md [hunk 2]
```diff
@@ -120,3 +120,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+## License
+
+Copyright © 2019-2020 Caleb Porzio and contributors
```
Comment (DanHarrin) [inline README.md:126]:
```suggestion
Copyright © Caleb Porzio
```

File: README.md [hunk 3]
```diff
@@ -120,3 +120,9 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
+
+Licensed under the MIT license, see [LICENSE.md](LICENSE.md) for details.
```
Comment (DanHarrin) [inline README.md:128]:
```suggestion
Laravel Livewire is open-sourced software licensed under the [MIT license](LICENSE.md).
```

**Comment by DanHarrin** [comment]
Just suggested a few changes based on the feedback on this thread @muzafferdede, what are your thoughts?

**Comment by muzafferdede** [comment]
> Just suggested a few changes based on the feedback on this thread @muzafferdede, what are your thoughts?

@DanHarrin I am not sure why @calebporzio has use License in Alpine JS like so since it's the same license. I think his thoughts will be more valuable than mine. Maybe there is some standards he might need follow, maybe it doesn't even matter. So let's wait a feedback from him before we think what is good.

**Comment by DanHarrin** [comment]
Sounds like a plan! I’ve checked a few other projects and the routes they’ve gone down with their licenses and it looks like we’re definitely on the right track. Let’s see what Caleb thinks.

**Comment by devcircus** [comment]
👍🏼

**Comment by calebporzio** [comment]
This looks great, thank you all

**Comment by carlosmintfan** [comment]
> I've never seen a license that included "contributors" in the copyright. I'm sure there are examples but I'd think it should just be Caleb. 🤷🏻‍♂️ Also, it would be good to include a section at the bottom of the Readme that links to the license file.

Sorry, no, I think you should include contributors. https://github.com/slimphp/Slim-Website/pull/710/commits/4aeba9c378139aac03080b342c04546ea3ec2d12.

**Pull request merged**